### DISCUSSION TITLE
How to handle future checkout dates?

### DISCUSSION THREAD

**Discussion by mmalik19** [snipe/snipe-it#15443] [Q&A]
Hi all,

I have been enjoying exploring what Snipe IT can do but have run into a hurdle. I have done various searches but can't seem to find much about the topic.

Our specific use case is: 

A bunch of kit that's available to be booked out by staff, currently done by outlook calendars but that's become very messy. Example: I want to check out a specific item between specific dates in the future. This doesn't seem to be possible with SnipeIT since it seems to default to current date when doing asset check outs and you can't select future dates.

Is this a limitation of the system?
How does everyone else that has Snipe IT in use handle scenarios where you want to assign assets to staff in future dates?

Thanks

**Comment by snipe** [comment]
> Is this a limitation of the system? 

It is, since if you're booking it out in the future, we don't know that it was ever actually booked out when it was scheduled.

**Reply by mmalik19** [reply]
Thanks for the reply. I wonder how other organisations that use SnipeIT are handling scenarios where assets are reserved for certain projects or workshops?
